lib/control/validation/fetch.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const HTTP_BAD_REQUEST = 400;
```

Comment:
I think we have a couple of constants for HTTP response codes scattered through the various validation modules. Do we want to consolidate those in a single place? I wouldn't try to do that in this pull request. Just an observation for a future refactor.